pkg/cmd/scan.go

```diff
@@ -180,41 +182,60 @@ func parseFromFlag(from []string) ([]config.SupplierConfig, error) {
 	for _, flag := range from {
 		schemePath := strings.Split(flag, "://")
 		if len(schemePath) != 2 || schemePath[1] == "" || schemePath[0] == "" {
-			return nil, fmt.Errorf(
-				"Unable to parse from flag: %s\nAccepted schemes are: %s",
+			return nil, errors.Wrapf(
+				cmderrors.NewUsageError(
+					fmt.Sprintf(
+						"\nAccepted schemes are: %s",
+						strings.Join(supplier.GetSupportedSchemes(), ","),
+					),
+				),
+				"Unable to parse from flag '%s'",
 				flag,
-				strings.Join(supplier.GetSupportedSchemes(), ","),
 			)
 		}
 
 		scheme := schemePath[0]
 		path := schemePath[1]
 		supplierBackend := strings.Split(scheme, "+")
 		if len(supplierBackend) > 2 {
-			return nil, fmt.Errorf(
-				"Unable to parse from scheme: %s\nAccepted schemes are: %s",
+			return nil, errors.Wrapf(
```

Comment:
I don't get why you have to wrap here when you create the error just inside the wrap ?